test/testdrive/protobuf.td

```diff
@@ -105,24 +105,56 @@ message SimpleId {
   string id = 1;
 }
 
-$ kafka-ingest topic=tester format=protobuf schema=${schema} message=simpleid publish=true
+$ kafka-ingest topic=simple format=protobuf schema=${schema} message=simpleid publish=true
 {"id": "a"}
 {"id": "b"}
 
-> CREATE MATERIALIZED SOURCE tester
-  FROM KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'testdrive-tester-${testdrive.seed}'
+> CREATE MATERIALIZED SOURCE simple
+  FROM KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'testdrive-simple-${testdrive.seed}'
   FORMAT PROTOBUF USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
 
-> SHOW COLUMNS FROM tester
+> SHOW COLUMNS FROM simple
 id         true   text
 mz_offset  false  bigint
 
-> SELECT * FROM tester
+> SELECT * FROM simple
 id   mz_offset
 ------------
 a    1
 b    2
 
+$ kafka-create-topic topic=imported
+
+$ set schema
+syntax = "proto3";
+
+import 'simple.proto';
+
+message Imported {
+    SimpleId id = 1;
+}
+
+$ kafka-publish-schema format=protobuf topic=imported ref-name=simple.proto ref-topic=testdrive-simple-${testdrive.seed}-value schema=${schema}
```

Comment:
So, I think you can avoid the special-caseyness of `kafka-publish-schema` if you just post at the CSR with exactly the references you want:

```
$ http-request method=POST url=${testdrive.schema-registry-url}/subjects/blah/versions content-type=application/json
{
    "schema": ${schema}
    "schemaType": "protobuf",
    "references": {
        // whatever
     }
}
```

Might as well leave the CCSR code in place though, because I'm sure we'll want that for Protobuf sinks some day!